### DISCUSSION TITLE
Errore device tracker

### DISCUSSION THREAD

**Discussion by Taz866** [indomus/forum#2761] [HUB: Home Assistant]
Buongiorno,
da circa un giorno HA non riconosce dov'è il mio cellulare, da lo stato come sconosciuto, inoltre ho notato un errore su device tracker in home assistant:

```
Logger: homeassistant.setup
Source: setup.py:332
First occurred: 09:35:47 (1 occurrences)
Last logged: 09:35:47

Error during setup of component device_tracker
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/setup.py", line 332, in _async_setup_component
    result = await task
             ^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/components/device_tracker/__init__.py", line 71, in async_setup
    await async_setup_legacy_integration(hass, config)
  File "/usr/src/homeassistant/homeassistant/components/device_tracker/legacy.py", line 199, in async_setup_integration
    tracker = await get_tracker(hass, config)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/components/device_tracker/legacy.py", line 521, in get_tracker
    devices = await async_load_config(yaml_path, hass, consider_home)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/components/device_tracker/legacy.py", line 1007, in async_load_config
    device.pop("vendor", None)
    ^^^^^^^^^^
AttributeError: 'bool' object has no attribute 'pop'
```

Non so come risolvere la cosa, potete aiutarmi? Grazie mille

**Comment by Taz866** [comment]
Risolto,
se qualcuno avesse lo stesso problema, avevo usato device tracker nello yaml, con l'aggiornamento di dicembre HA non lo supporta più e mi ha creato automaticamente gli stessi device nell'integrazione Ping (ICMP). Come consigliato da HA ho eliminato i miei traker_device dal file YAML ma dopo qualche ora non mi funzionavano più correttamente. 
Per risolvere ho dovuto eliminare le automazioni in cui avevo i tracker, tutti i tracker creati in automatico da HA e riavviato. In questo modo l'errore è sparito. Poi ho ricreato i tracker che mi servivano (mi serve per vedere quando pc o cellulare si collegano alla rete e fare delle automazioni con questo trigger).
Spero di essere stato abbastanza chiaro e che sarà di aiuto a qualcuno, buona giornata!

**Reply by snakuzzo** [reply]
Ciao.
Non occorreva specificarlo. E' già stato segnalato sia sulle note dela release ufficiale che da noi.

Chiudo
